spring-cloud-alibaba-examples/integrated-example/config-init/config/integrated-consumer.yaml

```diff
@@ -1,18 +1,20 @@
 spring:
-  datasource:
-    url: jdbc:mysql://integrated-mysql:3306/integrated_praise?useSSL=false&characterEncoding=utf8
   cloud:
     stream:
+      function:
+        definition: consumer;
       bindings:
-        praise-input:
+        consumer-in-0:
           destination: PRAISE-TOPIC-01
           content-type: application/json
           group: praise-consumer-group-PRAISE-TOPIC-01
       rocketmq:
         binder:
           name-server: rocketmq:9876
         bindings:
-          praise-input:
+          consumer-in-0:
             consumer:
               pullInterval: 4000
-              pullBatchSize: 4
\ No newline at end of file
+              pullBatchSize: 4
+  datasource:
+    url: jdbc:mysql://integrated-mysql:3306/integrated_praise?useSSL=false&characterEncoding=utf8
```

Comment:
Try to keep as few changes as possible, this part seems unnecessary to modify.

Reply:
This part of the change should be necessary, because the content of the Nacos configuration file after initialization was inconsistent with the content of the configuration file when it was not changed before. However, once the position is replaced, the content will be consistent after execution. I don't know why this is the case, but it can solve the problem. You can try it out